cfme/common/vm_views.py

```diff
@@ -545,15 +546,52 @@ class PolicySimulationView(BaseLoggedInPage):
     """
     Policy Simulation page for vms/instances
     """
+
+    title = Text("#explorer_title_text")
+
     @View.nested
     class form(View):  # noqa
-        policy = BootstrapSelect('policy_id')
+        policy_profile = BootstrapSelect("profile_id")
         # TODO policies table, ability to remove
         entities = View.nested(BaseNonInteractiveEntitiesView)
-        cancel_button = Button('Cancel')
+        cancel_button = Button("Cancel")
 
-    # TODO match quadicon
-    is_displayed = displayed_not_implemented
+    @property
+    def quadicon(self):
+        return self.form.entities.get_entity(name=self.context["object"].name)
+
+    @property
+    def is_displayed(self):
+        expected_title = "{} Policy Simulation".format(self.context["object"].VM_TYPE)
+        return self.title.text == expected_title and self.quadicon.is_displayed
```

Comment:
Optional:
Here you can check at least one `entity` available or not than particular `VM`. as per your test its common view from `All` and `Summery` vm page. `VM` name you can check in test.